syncd/syncd_applyview.cpp

```diff
@@ -2414,6 +2451,13 @@ std::shared_ptr<SaiObj> findCurrentBestMatchForNextHopGroup(
 {
     SWSS_LOG_ENTER();
 
+    /*
+     * For next hop group, let's try find matching NHG which will be based on
+     * NHG set as SAI_ROUTE_ENTRY_ATTR_NEXT_HOP_ID in route_entry.  We assume
+     * that each class IPv4 and IPv6 will have different NGH, and each IP
+     * prefix will only be assigned to one NHG.
```

Comment:
this is true as v4/v6 route will point to v4/v6 next hop.

Reply:
what do you mean ?